roles/common/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Update service account with image pull secret
+  k8s:
+    apply: true
+    definition: "{{ lookup('template', 'pulp-operator-sa.yaml.j2') }}"
```

Comment:
so it would overwrite the current service account, right?

Reply:
Right, but only if the `image_pull_secrets` is defined and only the `imagePullSecrets` field from the service account will get updated (nothing else would be modified).